### ISSUE TITLE
FEAT: CSS Re-write (Bluma)

### ISSUE THREAD

**Issue by seanvree** [Monitorr/Monitorr#84]
**Feature suggestion:**  Convert from bootstrap to Bluma

**Status**: Early development

**Current Branch**: https://github.com/Monitorr/Monitorr/tree/bluma

**Progress**:

- [x] (in-dev) working branch published:  https://github.com/Monitorr/Monitorr/tree/bluma
- [ ] (pending) Functionality
- [ ] (Pending) UI Desktop browsers:  x
- [ ] (Pending) UI Android: x
- [ ] (Pending) UI iOS: x
- [ ] (Pending) Pushed to DEV
- [ ] (Pending) Merged with MASTER
- [ ] (Pending) Release published

To Do:

- [ ] Move  clock to top left
- [ ] Add overview to top right?  ie: 0/4 services "offline" type of deal?
- [ ]  move refresh switch to bottom
- [ ]  Add manual curl/ping function feature to run a manual check of any service via simple input form with HOST: PORT which could be accessed by "drop down/radio button, etc (TBD)"

**Comment by wjbeckett** [comment]
There was already a branch setup for this. That's what the pearl_stag branch is. 
I had started down this path before Christmas. 

Won't just be a case rewrite either. Most of the code will be rewritten.

**Comment by seanvree** [comment]
@wjbeckett oh crap! I didn't know that!  All good, yeah, I didn't do MUCH work on it...was just messing around with it when I wanted to move the clock to the header...which lead me down a rabbit hole and figured out that we can't really do much with the clock UNTIL we do the CSS re-wrtie.  Too much to change, no point in doing it twice.  I created this branch just to capture other stuff that I had changed with respect to the clock.  

Yes, I agree about the code. I figured that out too.
